appveyor.yml

```diff
@@ -34,14 +36,24 @@ install:
 
 build_script:
   - CD %APPVEYOR_BUILD_FOLDER%
-  - script\build.cmd --code-sign --compress-artifacts
+  - IF [%TASK%]==[installer] (
+      IF [%APPVEYOR_REPO_BRANCH:~-9%]==[-releases] (
+        script\build.cmd --code-sign --compress-artifacts --create-windows-installer
+      ) ELSE (
+        ECHO Skipping installer and Atom build on non-release branch
+      )
+    ) ELSE (
+      ECHO Skipping installer build on non-installer build matrix row &&
```

Comment:
AppVeyor (or YAML?) smashes these [into a single line](https://ci.appveyor.com/project/Atom/atom/build/7566/job/pivaa9st2davrmpa#L428) before passing it off to cmd, so the `&&` is necessary for it to still be valid:

```
IF [%TASK%]==[installer] ( IF [%APPVEYOR_REPO_BRANCH:~-9%]==[-releases] ( script\build.cmd --code-sign --compress-artifacts --create-windows-installer ) ELSE ( ECHO Skipping installer and Atom build on non-release branch ) ) ELSE ( ECHO Skipping installer build on non-installer build matrix row && script\build.cmd --code-sign --compress-artifacts )
```